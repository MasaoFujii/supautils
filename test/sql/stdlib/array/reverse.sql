select supa.reverse(array['a', 'b', 'c']);

select supa.reverse(array[1, 2, 3]);

select supa.reverse(array[]::int[]);

select supa.reverse(null::int[]);
