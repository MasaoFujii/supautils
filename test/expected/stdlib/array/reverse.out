select supa.reverse(array['a', 'b', 'c']);
 reverse 
---------
 {c,b,a}
(1 row)

select supa.reverse(array[1, 2, 3]);
 reverse 
---------
 {3,2,1}
(1 row)

select supa.reverse(array[]::int[]);
 reverse 
---------
 
(1 row)

select supa.reverse(null::int[]);
 reverse 
---------
 
(1 row)

